bin/reth-bench/src/bench_mode.rs

```diff
@@ -0,0 +1,23 @@
+//! The benchmark mode defines whether the benchmark should run for a specific range of blocks or as
+//! a continuous stream of payloads.
```

Comment:
```suggestion
//! The benchmark mode defines whether the benchmark should run for a closed or open range of blocks. 
```
either way, the blocks, aka payloads, are being streamed continuously (actually discretely since by block number - uninterruptedly or sequentially would be better than 'continuously') from cl, the original sentence could make me wonder if the blocks are not streamed for the range mode